includes/class-wc-payments-account.php

```diff
@@ -486,7 +474,13 @@ private function get_oauth_return_url( $wcpay_connect_from ) {
 				],
 				admin_url( 'admin.php' )
 			)
-			: WC_Payment_Gateway_WCPay::get_settings_url();
+			: add_query_arg(
+				[
+					'page' => 'wc-admin',
+					'path' => '/payments/overview',
+				],
+				admin_url( 'admin.php' )
+			);
```

Comment:
@naman03malhotra that's the right place for defining redirect url after KYC. The only thing worth considering IMO is to move it to static function same as `get_settings_url`, maybe even for both pages, but it's up to you.

Reply:
It doesn't seem to change return URL when going to Stripe dashboard with "Edit details" link on the overview page or in account settings (like the "Terug naar WooCommerce" link on the picture).

<img width="391" alt="afbeelding" src="https://user-images.githubusercontent.com/683297/119827253-37f6fe00-bef9-11eb-877f-bc8f60f444a5.png">

Stripe URL which is used in those places is available under `wcpaySettings.accountLink`, worth checking if it has to be updated too.